SoftwareGuide/Latex/Architecture/SystemOverview.tex

```diff
@@ -378,27 +378,96 @@ \subsection{Multi-Threading}
 Multithreading is handled in ITK through a high-level design
```

Comment:
`Multi-threading`

for consistency. But I'm fine if we choose to remove the dash in **all** appearances.